### PR TITLE
Add composer.json

### PR DESCRIPTION
**Pull request by felixfbecker**
I found out that you can only install directly from GitHub with composer in root packages.
It would be nice if you could accept this PR, [submit the repository to packagist](https://packagist.org/packages/submit) and setup the GitHub webhook. Should only take a minute at max, after that it will automatically update the registry on every commit 🙂

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+    "name": "JetBrains/phpstorm-stubs",
```
Comment (hashar) [inline composer.json]: There is another JetBrains package published on packagist though it use all lower case for the company name: https://github.com/JetBrains/phpstorm-workshop/blob/master/composer.json done by @maartenba

    "name": "jetbrains/phpstorm-workshop"

Might want to stay consistent?

File: composer.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    "description": "PHP runtime & extensions header files for PhpStorm",
+    "homepage": "http://jetbrains.com/phpstorm",
```
Comment (hashar) [inline composer.json]: Switch the homepage URL to https: https://www.jetbrains.com/phpstorm/  They even have an Extended Validation certificate that shows up the company name

**Comment by felixfbecker** [comment]
@neuro159 wdyt?

**Comment by reedy** [comment]
Created a task over at YouTrack for visibility -- https://youtrack.jetbrains.com/issue/WI-34855

**Comment by hashar** [comment]
For info:

@rafaelcalleja forked the repo and added a [composer.json](https://github.com/rafaelcalleja/phpstorm-stubs/blob/master/composer.json). Published to Packagist as [phpstorm-stubs](https://packagist.org/packages/rc/phpstorm-stubs).

I am and @addshore are also going to publish a `wikimedia/phpstorm-stubs` based on https://github.com/wikimedia/mediawiki-libs-phpstorm-stubs/commit/e06a024014507ec40e003d720c46525d03681aed . Will remove it when this #149 lands and JetBrains publish the package.

**Comment by felixfbecker** [comment]
@hashar yeah, I so those, but I want to not worry about merging upstream changes.

**Comment by neuro159** [comment]
@felixfbecker can you please apply @hashar suggestions?..

**Comment by felixfbecker** [comment]
@neuro159 Done

**Comment by felixfbecker** [comment]
Awesome ❤️
Please ping me when you published to packagist and set up the GitHub hook :)

**Comment by hashar** [comment]
Packagist namespace `jetbrains/` is already existing due to https://packagist.org/packages/jetbrains/phpstorm-workshop  That has been uploaded by @maartenba  who I guess would need to submit https://github.com/JetBrains/phpstorm-stubs as well, then add other JetBrains developers/people to it.

**Comment by hashar** [comment]
Reached out to Maarten by email and he immediately replied he will handle it after the week-end :]

**Comment by felixfbecker** [comment]
Great. We still need a GitHub hook though, that has to be done by someone with admin rights to the repo afaik.

**Comment by maartenba** [comment]
Thanks! Just submitted it, the web hook is something still to be done (chasing the right people for that now)

**Comment by addshore** [comment]
@maartenba Thanks!

**Comment by neuro159** [comment]
Shall all be working now.

**Pull request merged**